templates/api.schema.yaml

```diff
@@ -0,0 +1,58 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Panda Sky CloudFormation Config
+description: Config for generating a Panda Sky CloudFormation template
+
+type: object
+required:
+  - name
+  - env
+  - projectID
+  - policyName
+  - rootResource
+  - resources
+  - environmentVariables
+  - aws
+properties:
+  name:
+    type: string
+  env:
+    type: string
+  projectID:
+    type: string
+  policyName:
+    type: string
+  rootResource: {$ref: "#/definitions/resource"}
+  resources:
+    type: object
+    additionalProperties: {$ref: "#/definitions/resource"}
+  schema:
+    type: object
+  aws:
+    type: object
+
+definitions:
+  refOrFn:
+    description: Verbatim YAML that defines either a 'Ref' or a 'Fn::GetAtt'
+    type: string
+  resource:
+    type: object
+    properties:
+      parentID: {$ref: "#/definitions/refOrFn"}
+      gateway:
```

Comment:
it contains names and IDs and other things specific to API Gateway.

Probably it's just transitional, useful only while I tease out and refactor.